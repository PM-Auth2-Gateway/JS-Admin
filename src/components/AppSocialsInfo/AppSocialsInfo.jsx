import { Row, Col } from 'react-bootstrap';
import SocialsList from '../SocialsList/SocialsList';

const AppSocialsInfo = ({ socials, loading, error }) => {
  return (
    <Row>
      <Col>
        <h4>Social networks</h4>
      </Col>
      <Col>
        <SocialsList list={socials} loading={loading} error={error} />
      </Col>
    </Row>
  );
};

export default AppSocialsInfo;
