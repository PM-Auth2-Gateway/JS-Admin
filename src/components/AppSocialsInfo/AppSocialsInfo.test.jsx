import { shallow } from 'enzyme';
import AppSocialsInfo from './AppSocialsInfo';

describe('AppSocialsInfo Component', () => {
  it('should render component', function () {
    const component = shallow(<AppSocialsInfo />);

    expect(component).toMatchSnapshot();
  });
});
