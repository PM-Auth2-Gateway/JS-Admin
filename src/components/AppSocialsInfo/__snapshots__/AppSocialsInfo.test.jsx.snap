// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppSocialsInfo Component should render component 1`] = `
<Row
  noGutters={false}
>
  <Col>
    <h4>
      Social networks
    </h4>
  </Col>
  <Col>
    <SocialsList />
  </Col>
</Row>
`;
